projectDocs/issues/triage.md

```diff
@@ -143,13 +143,22 @@ An issue is triaged if it is ready to be worked on.
 New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
 Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
 A `triaged` issue should also have a [priority label](#priority).
-Generally these labels are provided by NV Access only, particularly for issues that are controversial, or priority is unclear.
 
-For controversial changes, a product decision from NV Access may be required before applying the `triaged` label.
-This can be indicated with adding the label `blocked/needs-product-decision`.
+Community members should avoid adding a `triaged` label to feature requests or other decisions potentially involving significant or controversial changes to NVDA features or functionality, which may require community input or approval from NV Access.
+The label should also be avoided for other issues that are controversial, or where the priority is unclear, such as bug fixes with an unclear solution.
+For changes where a product decision from NV Access is required before applying the `triaged` label, the label `blocked/needs-product-decision` should be used.
+Community members should apply the `triaged` label where an issue is:
+
+* A well formed bug report following the issue template
+* Which can easily be understood
+* Isn't missing any important debug information
+* Can clearly be reproduced
+* Can clearly be prioritised
+* Doesn't require a controversial solution
 
 If an issue has been checked by NV Access, and needs further triage, the `needs-triage` label will be applied.
 Please notify NV Access when you believe the issue is ready for the `triaged` label.
+You can do this by tagging @nvaccess/developers or emailing info@nvaccess.org.
 
 A `triaged` issue that requires a complex fix may require advice from NV Access, such as a project plan, before implementation is started.
```

Comment:
This is more relevant to those planning on turning a triaged issue into a PR.
The idea is that if you see the "triaged" label, while it is ready to be worked on, it might be best to write up a proposed project plan before you start coding.